Puppetfile

```diff
@@ -27,6 +27,7 @@ mod 'saz/sudo', '3.1.0'
 mod 'saz/timezone'
 mod 'covermymeds-pgbouncer', '0.3.3'
 
+
```

Comment:
Accidentally committed whitespace?

Reply:
yep, we will fix that, it was leftover from trying to include the puppet module directly but failed due to dependencies conflicts.